metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1424,6 +1686,17 @@ def emit_sheets_as_charts(
             self.report.report_workunit(wu)
             yield wu
 
+    def _get_project_browse_path(self, project_luid: str) -> str:
```

Comment:
_get_project_browse_path_name

_project_luid_to_browse_path_name --> luid, project name suitable for browse path